src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx

```diff
@@ -37,7 +37,10 @@ export default function VideoVisitLocation({ appointment, facility }) {
     kind !== VIDEO_TYPES.clinic &&
     kind !== VIDEO_TYPES.gfe;
 
-  if (appointment.vaos.isPastAppointment && kind === VIDEO_TYPES.clinic) {
+  if (
+    appointment.vaos.isPastAppointment &&
+    (kind === VIDEO_TYPES.clinic || isAtlas)
```

Comment:
I don't think this is correct, it should be showing the ATLAS location, not the VA location. There's an AtlasLocation component further down in this component.

Reply:
🤦 Yes, of course! This was just a test... nothing to see here! 🙈

Reply:
Fixed 
<img width="669" alt="image" src="https://user-images.githubusercontent.com/8315447/120843053-a2342200-c53b-11eb-8d42-91c3684de0aa.png">